core/domain/role_services.py

```diff
@@ -108,6 +129,17 @@
 }
 
 
+def get_human_readable_role(role):
+    """Converts role to human readable format.
+
+    Args:
+        role: str. The role to convert.
+    Returns:
+        str. Role in human readable format.
+    """
+    return role.lower().replace('_', ' ')
```

Comment:
Yes, I think so, because there shouldn't be a programmatic link between the role IDs and how they are described. We should be able to change the latter freely without changing the former.